_component_design/breadcrumbs.md

```diff
@@ -54,9 +53,7 @@ Sometimes we can get so deep in an interface we lose track of where we are. Whil
 {% column left:"col-md-4" %}
 ## Naming and Abbreviations
 
-* The items in the breadcrumbs should have the same name as the page to which they point, another reason why page names should be short and descriptive.
-
-* However, sometimes we have really long page titles. Don’t let that weigh you down. The advantage of breadcrumbs is that we can use pages higher in the heirarchy, ie: “Billing” to avoid having to say “Billing Overview” in a more deeply nested page. So instead of repeating “Billing” in “Billing Overview”, just call it “Overview”. Keep it tight!
+* Ensure the items in the breadcrumbs have the same name as the page it navigates to. If the page title is longer than necessary, use shortened variations of the title. For example, instead of `Billing overview`, use `Overview`.
```

Comment:
@technabors  - This section essentially repeats lines 21 - 25. I would recommend avoiding duplication by either removing the 'few things to keep in mind' or linking the two.

Reply:
Removal is fine in my book as the text seems redundant.